lib/message_format.dart

```diff
@@ -253,8 +253,6 @@ class MessageFormat {
       var patternValue = currentPattern._value;
       var patternType = currentPattern._type;
 
-      _checkAndThrow(patternType is _BlockType,
```

Comment:
Why these code changes?

Reply:
`patternType` always has type `_BlockType`, and with null safety cannot be nullable. So, the condition is always true.